manual/SUMMARY.md

```diff
@@ -23,7 +25,7 @@
     - [genann](examples/genann/README.md)
     - [lil](examples/lil/README.md)
     - [xzoom](examples/xzoom/README.md)
-    - [robotfindskitten](examples/robotfindskitten/README.md)
+    <!-- - [robotfindskitten](examples/robotfindskitten/README.md) -->
```

Comment:
I noticed this wasn't tested by `scripts/test_examples.py` in the c2rust repo. What's the status of this example?

Reply:
The example was primarily for testing the refactoring engine, which is now deprecated.